docs/synchronization.md

```diff
@@ -4,13 +4,12 @@
 
 ## Introduction
 
```

Comment:
```suggestion
```

The "Introduction" heading is redundant as well; I've removed it in some places already (#13110 and I think some others too)